cloud/ovirt/ovirt_host_networks.py

```diff
@@ -0,0 +1,364 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Red Hat, Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+try:
+    import ovirtsdk4 as sdk
+    import ovirtsdk4.types as otypes
+except ImportError:
+    pass
+
+from ansible.module_utils.ovirt import *
+
+
+DOCUMENTATION = '''
+---
+module: ovirt_host_networks
+short_description: Module to manage host networks in oVirt
+version_added: "2.3"
+author: "Ondra Machacek (@machacekondra)"
+description:
+    - "Module to manage host networks in oVirt."
+options:
+    name:
+        description:
+            - "Name of the the host to manage networks."
+        required: true
+    state:
+        description:
+            - "Should the host be present/absent."
+        choices: ['present', 'absent']
+        default: present
+    bond:
+        description:
+            - "Dictionary describing network bond:"
+            - "C(name) - Bond name."
+            - "C(mode) - Bonding mode."
+            - "C(interfaces) - List of interfaces to create a bond."
+    interface:
+        description:
+            - "Name of the network interface where logical network should be attached."
+    networks:
+        description:
+            - "List of dictionary describing networks to be attached to interface or bond:"
+            - "C(name) - Name of the logical network to be assigned to bond or interface."
+            - "C(boot_protocol) - Boot protocol one of the I(none), I(static) or I(dhcp)."
+            - "C(address) - IP address in case of I(static) boot protocol is used."
+            - "C(prefix) - Routing prefix in case of I(static) boot protocol is used."
+            - "C(gateway) - Gateway in case of I(static) boot protocol is used."
+            - "C(version) - IP version. Either v4 or v6."
+    labels:
+        description:
+            - "List of names of the network label to be assigned to bond or interface."
+    check:
+        description:
+            - "If I(true) verify connectivity between host and engine."
+            - "If after changing the networks configuration the connectivity from host to the engine
+               is lost changes are rolled back."
```

Comment:
I'd rephrase:

Network configuration changes will be rolled back if connectivity between engine and the host is lost after changing network configuration.